nemoguardrails/library/autoguard/README.md

```diff
@@ -261,39 +261,38 @@ rails:
                   }
     input:
         flows:
-            - call autoguard input
+            - autoguard check input
     output:
         flows:
-            - call autoguard output
+            - autoguard check output
 ```
 We also have to add the autoguard's endpoint in parameters.
 
-One of the advanced configs is matching score (ranging between 0 to 1) which is a threshold that determines whether the guardrail will block the input/output or not.
+One of the advanced configs is matching score (ranging between 0 and 1) which is a threshold that determines whether the guardrail will block the input/output or not.
```

Comment:
i think the matching score is between 0 to 1, not just 0 and 1

Reply:
resolved in further commits